java/java.source.base/test/unit/src/org/netbeans/api/java/source/gen/LiteralTest.java

```diff
@@ -168,6 +168,87 @@ public void run(WorkingCopy workingCopy) throws java.io.IOException {
         assertEquals(golden, res);
     }
 
+    public void testTextBlocksNew() throws Exception {
+        testFile = new File(getWorkDir(), "Test.java");
+        TestUtilities.copyStringToFile(testFile,
+            "package hierbas.del.litoral;\n" +
+            "\n" +
+            "public class Test {\n" +
+            "    public static final String C;\n" +
+            "}\n"
+            );
+        String golden =
+            "package hierbas.del.litoral;\n" +
+            "\n" +
+            "public class Test {\n" +
+            "    public static final String C = \"\"\"\n" +
+            "                                   line \"1\n" +
+            "                                   line\\\"\\\"\\\"\"\"2\"\n" +
+            "                                   line\\n3\"\"\";\n" +
+            "}\n";
+        JavaSource testSource = JavaSource.forFileObject(FileUtil.toFileObject(testFile));
+        Task<WorkingCopy> task = new Task<WorkingCopy>() {
```

Comment:
Jackpot:
warning: This anonymous inner class creation can be turned into a lambda expression.